src/google/protobuf/compiler/cpp/cpp_bootstrap_unittest.cc

```diff
@@ -183,6 +183,69 @@ TEST(BootstrapTest, GeneratedFilesMatch) {
   }
 }
 
+//test Generate in cpp_generator.cc
+TEST(BootstrapTest, optionNotExist) {
+  CodeGenerator *generator=new cpp::CppGenerator();
```

Comment:
I think this line is causing a memory leak since the generator never gets deleted. You can just put the `CppGenerator` on the stack to solve that problem.

Reply:
Thank you for reminding， I will fix this

Reply:
Done!